authors/apps/authentication/renderers.py

```diff
@@ -1,25 +1,27 @@
 import json
 
 from rest_framework.renderers import JSONRenderer
-from .token import generate_jwt_token
 
 class UserJSONRenderer(JSONRenderer):
     charset = 'utf-8'
 
     def render(self, data, media_type=None, renderer_context=None):
-        # If the view throws an error (such as the user can't be authenticated
-        # or something similar), `data` will contain an `errors` key. We want
-        # the default JSONRenderer to handle rendering errors, so we need to
-        # check for this case.
+        """
+        If the view throws an error (such as the user can't be authenticated
+        or something similar), `data` will contain an `errors` key. We want
+        the default JSONRenderer to handle rendering errors, so we need to
+        check for this case.
+        """
         errors = data.get('errors', None)
 
 
         if errors is not None:
```

Comment:
@meshack-mbuvi this line is quite repetitive use `if errors` because in python None is considered falsy.